tests/scalers/selenium-grid-test.ts

```diff
@@ -0,0 +1,626 @@
+            - configMapRef:
+                name: selenium-event-bus-config
+          ports:
+            - containerPort: 5553
+              protocol: TCP
+          volumeMounts:
+            - name: dshm
+              mountPath: /dev/shm
+          resources:
+            limits:
+              cpu: "1"
+              memory: 1Gi
+            requests:
+              cpu: "1"
+              memory: 1Gi
+      volumes:
+        - name: dshm
+          emptyDir:
+            medium: Memory
+            sizeLimit: 1Gi
+---
+# Source: selenium-grid/templates/chrome-node-91-service.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: selenium-chrome-node-91
+  namespace: {{NAMESPACE}}
+  labels:
+    name: selenium-chrome-node-91
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/instance: selenium-hpa
+    app.kubernetes.io/version: 4.0.0-beta-1-prerelease-20210114
+    app.kubernetes.io/component: selenium-grid-4.0.0-beta-1-prerelease-20210114
+    helm.sh/chart: selenium-grid-0.2.0
+spec:
+  type: ClusterIP
+  selector:
+    app: selenium-chrome-node-91
+  ports:
+    - name: tcp-chrome
+      protocol: TCP
+      port: 6900
+      targetPort: 5900
+---
+# Source: selenium-grid/templates/chrome-node-91-hpa.yaml
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: selenium-grid-chrome-91-scaledobject
+  namespace: {{NAMESPACE}}
+  labels:
+    deploymentName: selenium-chrome-node-91
+spec:
+  maxReplicaCount: 8
+  scaleTargetRef:
+    name: selenium-chrome-node-91
+  triggers:
+    - type: selenium-grid
+      metadata:
+        url: '{{SELENIUM_GRID_GRAPHQL_URL}}'
+        browserName: 'chrome'
+        browserVersion: '91.0'
+---`
+
+const scaledObjectYaml = `---
+# Source: selenium-grid/templates/chrome-node-hpa.yaml
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: selenium-grid-chrome-scaledobject
+  namespace: {{NAMESPACE}}
+  labels:
+    deploymentName: selenium-chrome-node
+spec:
+  maxReplicaCount: 8
+  scaleTargetRef:
+    name: selenium-chrome-node
+  triggers:
+    - type: selenium-grid
+      metadata:
+        url: '{{SELENIUM_GRID_GRAPHQL_URL}}'
+        browserName: 'chrome'
+---
+# Source: selenium-grid/templates/firefox-node-hpa.yaml
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: selenium-grid-firefox-scaledobject
+  namespace: {{NAMESPACE}}
+  labels:
+    deploymentName: selenium-firefox-node
+spec:
+  maxReplicaCount: 8
+  scaleTargetRef:
+    name: selenium-firefox-node
+  triggers:
+    - type: selenium-grid
+      metadata:
+        url: '{{SELENIUM_GRID_GRAPHQL_URL}}'
+        browserName: 'firefox'`
+
+const seleniumGridTestsYaml = `apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: {{JOB_NAME}}
+  name: {{JOB_NAME}}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{JOB_NAME}}
+    spec:
+      containers:
+      - name: {{CONTAINER_NAME}}
+        image: prashanth0007/selenium-random-tests:v1.0.2
```

Comment:
@prashanth-volvocars  can you add the code for the test image here https://github.com/kedacore/test-tools/tree/master/e2e/images

Reply:
Created a pull request at https://github.com/kedacore/test-tools/pull/15